flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdFilteredColumnInterval.scala

```diff
@@ -236,7 +248,9 @@ class FlinkRelMdFilteredColumnInterval private extends MetadataHandler[FilteredC
       mq: RelMetadataQuery,
       columnIndex: Int,
       filterArg: Int): ValueInterval = {
-    checkArgument(filterArg == -1)
+    if (filterArg != -1) {
```

Comment:
please add a test case to cover this change

Reply:
convered by the newly added case `testGetColumnIntervalOnAggregateWithFilter`